docs/markdown/Qt5-module.md

```diff
@@ -38,3 +39,10 @@ executable('myprog', 'main.cpp', 'myclass.cpp', moc_files,
 
 The 'modules' argument is used to include Qt modules in the project.
 See the Qt documentation for the [list of modules](http://doc.qt.io/qt-5/qtmodules.html).
+
+## private headers (since v0.47.0)
+
+Some projects needs Qt's private headers to build, that's why a **private_headers** keyword argument has been added to [dependency](Reference-manual.md#dependency) method.
```

Comment:
I'm not sure this is phrased in the best way for what's intended to be reference documentation.  Perhaps just:

```
Setting the **private_headers** keyword argument (since 0.47.0) to true will add the private include path of the given module to the compiler flags.
```